Gemfile

```diff
@@ -232,4 +232,7 @@ group :development do
 
   # TomDoc for YARD (http://rubyworks.github.com/yard-tomdoc)
   gem "yard-tomdoc"
+
+  gem "dotenv-rails"
```

Comment:
What's this being used for?

Reply:
Allows for use of ENV variables which many people like to do to keep their code and dev. environment separate. For example, testing each DB_ADAPTER as described in `database_example.yml`

``` YML
defaults: &defaults
  adapter: <%= ENV['DB_ADAPTER'] || 'postgresql' %>
  encoding: <%= ENV['DB_ADAPTER'] == "mysql2" ? "utf8mb4" : "" %>
  username: <%= ENV["DB_ADAPTER"] == "postgresql" ? 'postgres' : '' %>
  database: roadmap_<%= ENV['RAILS_ENV'] %>
  pool: 5

development:
  <<: *defaults

test:
  <<: *defaults

```